src/browser.ts

```diff
@@ -60,11 +61,20 @@ export interface AnalyticsBrowserSettings extends AnalyticsSettings {
    * for the source.
    */
   cdnSettings?: LegacySettings & Record<string, unknown>
+  /**
+   * If provided, will override the default Segment CDN (https://cdn.segment.com) for this application.
+   * Typically used to proxy client-side requests from analytics.js through a specific server.
```

Comment:
`client-side requests` makes me think of requests hitting the tracking API which this doesn't actually include. It's just for requests that load source settings and the dynamically loaded pieces of the library.

Maybe something like:
> Typically used to proxy Source settings and analytics.js...